ui/apps/platform/src/Containers/Violations/Details/Deployment/DeploymentOverview.tsx

```diff
@@ -20,7 +22,14 @@ function DeploymentOverview({
 }: DeploymentOverviewProps): ReactElement {
     return (
         <DescriptionList isCompact isHorizontal>
-            <DescriptionListItem term="Deployment ID" desc={alertDeployment.id} />
+            <DescriptionListItem
+                term="Deployment ID"
+                desc={
+                    <Link to={`${vulnManagementPath}/deployment/${alertDeployment.id}`}>
```

Comment:
Aren't there path / url concatenation operators to avoid double or missing slashes?

Reply:
There are, if we defined patterns for all our paths, but we don't currently. I chose to follow the string template pattern we use throughout the app, because I think consistency is better than different patterns in different places in the code.

Reply:
> I think consistency is better than different patterns in different places in the code

I think so too. Resolving.